### DISCUSSION TITLE
绳网都有了，不妨大胆一点，人工智能Fairy还会远吗？

### DISCUSSION THREAD

**Discussion by bai-yixing** [share121/inter-knot#2596] [其他]
![200fc52420674ffd8939c51852dd93d8](https://github.com/user-attachments/assets/1b2259b0-1fe1-4d5b-8b6a-3caa44b87a31)

**Comment by 1luik** [comment]
你可以尝试自己训练
rwkv：https://www.bilibili.com/read/cv26232425/
matmut free的源码自己看着调我懒得给你做教程：https://github.com/ridgerchu/matmulfreellm
Transformer多模态：https://zhuanlan.zhihu.com/p/687887823
Transformer简单版：https://blog.csdn.net/weixin_43564920/article/details/135655475
多模态的实现方式：https://new.qq.com/rain/a/20240702A00UTI00
pytorch教程给我就有用机翻插件：https://pytorch.org/

**Reply by bai-yixing** [reply]
谢谢大佬！

**Reply by ancheers** [reply]
B站UP主“TIG-幻日”已经做出一个了fairy了呀，虽然没那么智能但是做个代餐还行吧。（我跟她说我是用说话的方式跟她聊天，她那犟驴非说自己是文字交流，估计就是用语音转文字跟AI沟通的）
